client/my-sites/email/email-management/list/email-management-list-item.jsx

```diff
@@ -0,0 +1,358 @@
+import hasLoadedGSuiteUsers from 'calypso/state/selectors/has-loaded-gsuite-users';
+import MaterialIcon from 'calypso/components/material-icon';
+import VerticalNavItem from 'calypso/components/vertical-nav/item';
+import QueryEmailForwards from 'calypso/components/data/query-email-forwards';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+class EmailManagementListItem extends React.Component {
+	static propTypes = {
+		currentRoute: PropTypes.string.isRequired,
+		domain: PropTypes.object.isRequired,
+		emails: PropTypes.array.isRequired,
+		selectedSite: PropTypes.object.isRequired,
+		user: PropTypes.object.isRequired,
+		userCanManageOptions: PropTypes.bool.isRequired,
+	};
+
+	onAddTitanMailbox = () => {
+		this.recordAddEmailEvent( 'titan' );
+
+		const { currentRoute, domain, selectedSite } = this.props;
+		page( emailManagementNewTitanAccount( selectedSite.slug, domain.name, currentRoute ) );
+	};
+
+	onAddGoogleUser = () => {
+		this.recordAddEmailEvent( 'google' );
+
+		const { currentRoute, domain, selectedSite } = this.props;
+		page( emailManagementAddGSuiteUsers( selectedSite.slug, domain.name, currentRoute ) );
+	};
+
+	onAddEmailForward = () => {
+		this.recordAddEmailEvent( 'email-forward' );
+
+		const { domain, selectedSite } = this.props;
+		page( domainManagementEmailForwarding( selectedSite.slug, domain.name ) );
+	};
+
+	recordAddEmailEvent( accountType ) {
+		const { domain } = this.props;
+		recordTracksEvent( 'calypso_email_management_add_email_account', {
+			domain: domain.name,
+			account_type: accountType,
+		} );
+	}
+
+	canUserAddEmailAccounts( domain ) {
+		const { user } = this.props;
+
+		if ( hasTitanMailWithUs( domain ) ) {
+			// TODO: Remove this check once the back end support for this is in place.
+			const titanOwnerId = getTitanMailOwnerId( domain );
+			if ( titanOwnerId === null ) {
+				return true;
+			}
+			return titanOwnerId === user.ID;
+		}
+
+		if ( hasGSuiteWithUs( domain ) ) {
+			return getGSuiteOwnerId( domain ) === user.ID;
+		}
+
+		if ( hasEmailForwards( domain ) ) {
+			/* TODO: Update the logic */
+			return this.props.userCanManageOptions;
+		}
+
+		return false;
+	}
+
+	getHeaderImage( domain ) {
+		const { translate } = this.props;
+
+		if ( hasTitanMailWithUs( domain ) ) {
+			return (
+				<Gridicon
+					className="email-management-list-item__header-icon titan"
+					icon="my-sites"
+					size={ 36 }
+				/>
+			);
+		}
+
+		if ( hasGSuiteWithUs( domain ) ) {
+			return (
+				<img
+					className="email-management-list-item__header-icon google"
+					src={ googleWorkspaceIcon }
+					alt={ translate( 'Google Workspace icon' ) }
+				/>
+			);
+		}
+
+		if ( hasEmailForwards( domain ) ) {
+			return (
+				<img
+					className="email-management-list-item__header-icon email-forward"
+					src={ emailForwardingIcon }
+					alt={ translate( 'Email Forwarding icon' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **31** times:
`translate( 'Email Forwarding' )` *ES Score: 7*